spec/locales/en/translation.json

```diff
@@ -50,6 +50,7 @@
     "load": "Load",
     "logo": "Logo",
     "metadataTooltip": "View information/metadata about this object",
+    "mirrorTooltip": "Mirror image",
```

Comment:
Whoops, didn't see that in slack.  I don't even see the files being used in the tests anywhere.  Am I missing any?